src/main/drivers/pwm_output.h

```diff
@@ -30,11 +30,27 @@ typedef enum {
     PWM_TYPE_DSHOT600,
     PWM_TYPE_DSHOT300,
     PWM_TYPE_DSHOT150,
+    PWM_TYPE_DSHOT900,
+    PWM_TYPE_DSHOT1200,
     PWM_TYPE_MAX
```

Comment:
We should bear the pain now, rather than forever have to live with an out of order list.